server/server.go

```diff
@@ -541,6 +658,9 @@ func (server *HTTPServer) addEndpointsToRouter(router *mux.Router) {
 	router.HandleFunc(apiPrefix+DislikeRuleEndpoint, server.dislikeRule).Methods(http.MethodPut, http.MethodOptions)
 	router.HandleFunc(apiPrefix+ResetVoteOnRuleEndpoint, server.resetVoteOnRule).Methods(http.MethodPut, http.MethodOptions)
 	router.HandleFunc(apiPrefix+ClustersForOrganizationEndpoint, server.listOfClustersForOrganization).Methods(http.MethodGet)
+	router.HandleFunc(apiPrefix+DisableRuleForClusterEndpoint, server.disableRuleForCluster).Methods(http.MethodPut, http.MethodOptions)
+	router.HandleFunc(apiPrefix+EnableRuleForClusterEndpoint, server.enableRuleForCluster).Methods(http.MethodPut, http.MethodOptions)
+	router.HandleFunc(apiPrefix+ListDisabledRulesForClusterEndpoint, server.listDisabledRulesForCluster).Methods(http.MethodGet)
```

Comment:
Please, instead of using direct endpoint to disabled rules, better add query `disabled` to `ReportEndpoint`. It will simplify filtering. Without `disabled` query let it return all rules.

Reply:
Agree. 

And also according to the mock https://marvelapp.com/852jaj9/screen/68494249 it looks like we would need a way to fetch 1 all rules, 2 disabled rules and 3 enabled rules. But also all of this could be done on frontend the same way as we did for another filters.

Reply:
Are you guys referring the List endpoint? I was using it a lot when I was testing and I left it included in the final endoints.go by mistake (I'll remove it). I didn't want to delete it completely because I think it can be used for some tests.

Regarding the filtering - I assumed that this would be done on frontend like all the other filtering and sorting and considering that the design/mockup is missing information about what is default (I guess Enabled?), and since the "Status" is just yet another filter, I don't think it wouldn't make sense for it to behave differently from other filters - reload (and wait for) the whole results just for one keyword. 

@bond95 @Sergey1011010

Reply:
@Bee-lee then this endpoint should be included only in debug mode. Thanks for clarification, you're right, filtering can be done on frontend.

Reply:
> Are you guys referring the List endpoint? I was using it a lot when I was testing and I left it included in the final endoints.go by mistake (I'll remove it). I didn't want to delete it completely because I think it can be used for some tests.


@Bee-lee yes, endpoint with query `disabled` could be used for tests same as this endpoint, but without additional endpoint.

> Regarding the filtering - I assumed that this would be done on frontend like all the other filtering and sorting and considering that the design/mockup is missing information about what is default (I guess Enabled?), and since the "Status" is just yet another filter, I don't think it wouldn't make sense for it to behave differently from other filters - reload (and wait for) the whole results just for one keyword.

Yes, filtering can be done on frontend, but I still don't see any reason to use separate endpoint. I prefer to have endpoint which can return enabled or disabled or both rules according to query, because it's more flexible.